11-type-constructors/.11-type-constructors.tex

```diff
@@ -0,0 +1,13 @@
+\documentclass[12pt]{article}
```

Comment:
this file should be deleted I think.

Reply:
It's to generate the markdown file
I can delete it since it seems like it's confusing